.github/workflows/test.yml

```diff
@@ -10,6 +10,10 @@ on:  # yamllint disable-line rule:truthy
         default: ''  # if not provided: When checking out the repository that triggered a workflow, this defaults to the reference or SHA for that event. Otherwise, uses the default branch.
       eve_image:
         type: string
+      eve_log_level:
+        type: string
+        required: false
+        default: 'info'
```

Comment:
Before using this from the eve repo, we should try to set 'debug' first here.
We need to check how large the output artifacts will be and whether they are not exceeding some github limits.

Reply:
I couldn't find any official documentation on the topic, but there is a [rumor](https://github.com/actions/upload-artifact/issues/9) that the limit is 5 GB per file.

Our artifacts still stay under 50 MB https://github.com/lf-edge/eden/actions/runs/11437464614?pr=1033

Reply:
![before](https://github.com/user-attachments/assets/6efb7436-6757-49dc-b70e-4c2fc8214d6a)
![after](https://github.com/user-attachments/assets/44ecbb20-b41e-4f9a-98cf-25a8ab6572d0)
Here are the before and after screenshots. I'd say the sizes stay comparable.

Reply:
Does it actually work?
When I look for example into https://github.com/lf-edge/eden/actions/runs/11437464614/artifacts/2081755978,
I do not see debug logs inside eve-info.tar.gz
`debug.default.loglevel` is configured to `info` in `persist-status/zedagent/ConfigItemValueMap/global.json`

Reply:
Sorry, I just noticed that as well :facepalm: 
Will fix very soon

Reply:
ok, now I think I got it right
![image](https://github.com/user-attachments/assets/f3308570-cb10-4d5a-81b7-43b97d689349)
the sizes have increased but still not dramatic

Reply:
maybe we can try running it like this for a while and if we reach some limit we can still revert it

Reply:
btw github copilot can provide some interesting insights that cannot be found in the github documentation
![image](https://github.com/user-attachments/assets/6609f5ce-e5d9-45ff-9105-7875003860e6)
I'm just wondering if it's true or hallucinations :smile: